src/api/app/models/staging_workflow.rb

```diff
@@ -20,10 +20,12 @@ def ignored_requests
 
   private
 
-  def init_staging_projects
-    return unless new_record?
+  def create_staging_projects
     ['A', 'B'].each do |letter|
-      staging_projects << Project.find_or_initialize_by(name: "#{project.name}:Staging:#{letter}")
+      p = Project.find_or_initialize_by(name: "#{project.name}:Staging:#{letter}")
```

Comment:
p? I save that one for later

Reply:
I have my reasons! if I would call it `project`, when accessing `project.name` in the same line it will fail. :unamused:

Reply:
Why not take n then, but the middle of *my* name?